### PR TITLE
book: Replace git version of the book with link to stable

### PR DESCRIPTION
**Pull request by Hofer-Julian**

### CODE REVIEW

File: git/book [hunk 1]
```diff
@@ -0,0 +1 @@
+/var/home/julian/Projekte/gnome/libraries/gtk4-rs/stable/latest/book
```
Comment (bilelmoussaoui) [inline git/book:1]: That is probably not gonna work :p

Comment (bilelmoussaoui) [reply git/book:1]: That is how I do it for the docs for each release
```
    - cd $repo
    - unlink ./stable/latest/docs
    - cd ./stable/latest
    - ln -sf ../$version/docs ./docs
 ```

Comment (Hofer-Julian) [reply git/book:1]: Mmmh weird, that's how I created the symbolic link as well.

Will have a look later.

Comment (Hofer-Julian) [reply git/book]: For some reason, I had to pass the `-r` flag for it to work.

Should be fine now.

**Pull request merged**